shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/BaseSimulateRequest.java

```diff
@@ -533,10 +541,12 @@ public void function(
                         .collect(
                             Collectors.toMap(
                                 Function.identity(),
-                                name ->
-                                    inputSource
-                                        .fetch(name, readStale)
-                                        .collect(Collectors.toList())));
+                                name -> {
+                                  CACHE_REFRESH.labels(readStale ? "stale" : "live", name).inc();
```

Comment:
It might be clearer to use the same labels as are in the UI: `cached` & `fresh`. I'm happy to make this change in another PR if you'd rather not.

Reply:
Fixed.